app/styles/_story-tease.scss

```diff
@@ -30,6 +30,16 @@
   @include mq($small-only) {
     width: 90%;
 
+    .flag {
+      display: flex;
+      align-items: center;
+      justify-content: center;
```

Comment:
Can you use the mixin instead of raw values? The mixin provides for crossbrowser compat